pubspec.yaml

```diff
@@ -12,7 +12,7 @@ environment:
   sdk: '>=2.6.0 <3.0.0'
 
 dependencies:
-  analyzer: ^0.39.9
+  analyzer: ^0.39.13
```

Comment:
Why such a big jump in version numbers?

Reply:
I believe that's when `SecurityLintCodeWithUniqueName` was introduced?

Reply:
Sorry. I was confused about what I was looking at. LGTM.